elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -213,7 +213,11 @@ export class PeerPool extends EventEmitter {
 		};
 		this._handleFailedPeerInfoUpdate = (error: Error) => {
 			// Re-emit the error to allow it to bubble up the class hierarchy.
-			this.emit(EVENT_FAILED_PEER_INFO_UPDATE, error);
+			this.emit(EVENT_FAILED_TO_FETCH_PEER_INFO, error);
+		};
+		this._handleFailedToCollectPeerDetails = (error: Error) => {
```

Comment:
same as above, breaking down this handler into two for `fetchStatus` and `fetchPeers`